inst/include/Rcpp/exceptions.h

```diff
@@ -89,6 +89,48 @@ namespace Rcpp {
             file_io_error("file already exists", file) {}	// #nocov end
     };
 
+    #define RCPP_ADVANCED_EXCEPTION_CLASS(__CLASS__, __WHAT__)                                                                                                                                                     \
```

Comment:
Enabling C++11 mode for `tinyformat` triggered a lot of error spam that is stuttering shell. So, I've left out this change until I can investigate the file a bit more after I grab some sleep.